File: Consigna3.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
#include <iomanip>
using namespace std;

int main() {
    srand(time(0));
    const int N = 25;
    string sucursales[N];
    int ventas[N];
    int suma = 0;
    for (int i = 0; i < N; i++) {
        sucursales[i] = "Sucursal " + to_string(i + 1);
        ventas[i] = rand() % 9001 + 1000; 
        suma += ventas[i];
    }
    double promedio = suma / (double)N;
    cout << fixed << setprecision(2);
    cout << "Promedio de ventas: " << promedio << endl;
    cout << "Sucursales con ventas mayores al promedio:" << endl;
    for (int i = 0; i < N; i++) {
        if (ventas[i] > promedio) {
            cout << sucursales[i] << ": " << ventas[i] << endl;
        }
    }
    return 0;
}

File: Consigna3.py
import random
sucursales = [f"Sucursal {i+1}" for i in range(25)]
ventas = [random.randint(1000, 5000) for _ in range(25)] 
promedio = sum(ventas) / len(ventas)
print("Ventas promedio:", promedio)
print("Sucursales con ventas por encima del promedio:")
for i in range(len(sucursales)):
    if ventas[i] > promedio:
        print(sucursales[i], "con ventas de", ventas[i])